recipes/pagmo2/pre_2.11/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
```

```suggestion
conan_basic_setup(KEEP_RPATHS)
```
Better support for reallocation on Mac

---

true, but doesn't really matter here since it's header only ^^